### ISSUE TITLE
[1224/4777] LINK ./zlib_bench FAILED: zlib_bench

### ISSUE THREAD

**Issue by lazyparser** [v8-riscv/v8-riscv-tools#10]
How to:

```
docker run --rm -ti ubuntu:18.04
# then paste the one-line
apt update ; apt -y install git htop build-essential python python-pip curl wget git lsb-release vim sudo ; \
git clone https://github.com/v8-riscv/v8-riscv-tools ; \
bash v8-riscv-tools/init_v8_all_in_one.sh
```

you will eventually face the error.

be aware that this scripts is not automatic. it expect the user have a few interactions.

```
cp -a /root/opt/riscv/riscv64-unknown-linux-gnu/lib* /root/opt/riscv/sysroot
mkdir -p stamps/ && touch stamps/build-gcc-linux-stage2
~/v8-riscv
Done. Made 221 targets from 96 files in 132ms
ninja: Entering directory `out/riscv64.native.debug'
[1224/4777] LINK ./zlib_bench
FAILED: zlib_bench
python "../../build/toolchain/gcc_link_wrapper.py" --output="./zlib_bench" -- riscv64-unknown-linux-gnu-g++ -Wl,--fatal-warnings -Wl,--build-id -fPIC -Wl,-z,noexecstack -Wl,-z,relro -Wl,-z,now -Wl,-z,defs -Wl,--as-needed -rdynamic -nodefaultlibs -pie -Wl,--disable-new-dtags -o "./zlib_bench" -Wl,--start-group @"./zlib_bench.rsp"  -Wl,--end-group  -lc -lgcc_s -lm -lrt -ldl -lpthread
/root/opt/riscv/lib/gcc/riscv64-unknown-linux-gnu/10.2.0/../../../../riscv64-unknown-linux-gnu/bin/ld: obj/buildtools/third_party/libc++/libc++/barrier.o: in function `.L95':
barrier.cpp:(.text._ZNSt3__124__barrier_algorithm_base8__arriveEh[_ZNSt3__124__barrier_algorithm_base8__arriveEh]+0x216): undefined reference to `__atomic_compare_exchange_1'
/root/opt/riscv/lib/gcc/riscv64-unknown-linux-gnu/10.2.0/../../../../riscv64-unknown-linux-gnu/bin/ld: obj/buildtools/third_party/libc++/libc++/barrier.o: in function `.L116':
barrier.cpp:(.text._ZNSt3__124__barrier_algorithm_base8__arriveEh[_ZNSt3__124__barrier_algorithm_base8__arriveEh]+0x362): undefined reference to `__atomic_compare_exchange_1'
/root/opt/riscv/lib/gcc/riscv64-unknown-linux-gnu/10.2.0/../../../../riscv64-unknown-linux-gnu/bin/ld: obj/buildtools/third_party/libc++/libc++/barrier.o: in function `.L145':
barrier.cpp:(.text._ZNSt3__124__barrier_algorithm_base8__arriveEh[_ZNSt3__124__barrier_algorithm_base8__arriveEh]+0x4be): undefined reference to `__atomic_compare_exchange_1'
collect2: error: ld returned 1 exit status
[1255/4777] CXX obj/third_party/inspector_protocol/crdtp_test/cbor_test.o
ninja: build stopped: subcommand failed.
root@e6a794d5209f:~#

```

**Comment by lazyparser** [comment]
Hopefully fixed by #11

**Comment by lazyparser** [comment]
Fixed.
